tests/misc/test_extra_group_pkgs_can_be_installed.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import pytest
+
+from lib.commands import SSHCommandFailed
+from packaging import version
+from subprocess import run, CalledProcessError
+
+def install_failed(pkg_name, h):
+    """Check if a package can be installed or not.
+
+    Returns True if the package cannot be installed, False otherwise.
+    It saves the yum state before trying to install a package and restore
+    it after.
+    """
+    failed = False
+    h.yum_save_state()
+
+    try:
+        h.yum_install([pkg_name])
+    except SSHCommandFailed:
+        failed = True
+
+    h.yum_restore_saved_state()
+    return failed
+
+def test_extra_group_packages_can_be_installed(host):
+    v = version.parse(host.inventory['PRODUCT_VERSION'])
+    url = f"https://reports.xcp-ng.org/{v}/extra_installable.txt"
+    logging.info(f"getting extra packages from {url}")
+
+    try:
+        ret = run(
+            ["wget", "-O-", "-q", url],
```

Comment:
As this is executed on the client side, we could simply use python to download the file. Using `urllib.request.urlopen` for example (https://stackoverflow.com/questions/7243750/download-file-from-web-in-python-3)